styles.css

```diff
@@ -1579,487 +1818,308 @@ footer {
-    height: 87px;
+  .Img-Placeholder {
+    background-size: contain;
+    width: 409px;
+    height: 319px;
   }
 
-  .project-2 > p {
-    width: 295px;
-    height: 87px;
+  .title-post {
+    width: 327px;
+    height: 48px;
+    margin-top: 0;
+    margin-bottom: 0;
   }
 
-  .project-2 > h1 {
-    width: 295px;
-    height: 81px;
+  .tag-1 {
+    margin-left: 0;
+    margin-right: 0;
   }
 
-  .project-3 > p {
-    width: 295px;
-    height: 87px;
+  .tag-2,
+  .tag-3,
+  .tag-4 {
+    margin-right: 0;
+    margin-left: 17px;
   }
 
-  .project-3 > h1 {
-    width: 295px;
-    height: 81px;
+  .work-grid {
+    width: 724px;
   }
 
-  .project-4 {
-    width: 327px;
-    height: 386px;
+  .card1 {
+    width: 756px;
+    height: 584px;
+    left: -44.5px;
   }
 
-  .project-5 {
-    width: 327px;
-    height: 386px;
-    top: 410px;
+  .card2 {
+    width: 756px;
+    height: 674px;
+    left: -44.5px;
   }
 
-  .project-6 {
-    width: 327px;
-    height: 386px;
-    top: 820px;
+  .work-section {
+    width: 781px;
+    left: 46px;
+    height: 1217px;
   }
 
-  .project-4 > h1 {
-    width: 295px;
-    height: 81px;
+  .about-myself-section {
+    top: 2370px;
   }
 
-  .project-4 > p {
-    width: 295px;
-    height: 87px;
+  .description {
+    width: 445px;
   }
 
-  .project-5 > p {
-    width: 295px;
-    height: 87px;
+  .about-me-frame {
+    height: 259px;
+    width: 250px;
   }
 
-  .project-5 > h1 {
-    width: 295px;
-    height: 81px;
+  .about-me-sec {
+    left: 46px;
   }
 
-  .project-6 > p {
-    width: 295px;
-    height: 87px;
+  .black-line {
+    width: 773px;
+    left: 48px;
   }
 
-  .project-6 > h1 {
-    width: 295px;
-    height: 81px;
+  .skill {
+    left: -59px;
+    justify-content: space-evenly;
+    flex-direction: row;
+    width: 921.46px;
   }
 
-  .tags {
-    padding-left: 16px;
-    margin-top: 20px;
+  .contact-form {
+    left: 33px;
+    height: 596px;
+    width: 785px;
+    justify-content: space-around;
+    top: 3565px;
   }
 
-  .button-p {
+  .contact-form h1 {
     width: 327px;
-    margin-top: 63px;
+    font-size: 30px;
   }
 
-  .nav-icon {
-    display: none;
+  .tags {
+    width: 253px;
   }
 
-  .navabar {
-    display: block;
-    margin-right: 20px;
+  input {
+    width: 240px;
   }
-}
-
-.project-1:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
-
-.project-2:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
 
-.project-3:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
+  textarea {
+    width: 240px;
+  }
 
-.project-4:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
+  form {
+    width: 268px;
+  }
 
-.project-5:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
+  .form {
+    margin: 0 0;
+    width: 273px;
+  }
 
-.project-6:hover {
-  background-image: url(./Desktop-images/Img_Placeholder.png);
-}
+  footer {
+    top: 4086px;
+  }
```

Comment:
- [ Optional ] Kindly remove commented out code so that your code base  be clean.